src/components/Input/index.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { letterSpacing, colors, size } from '../../utils'
+
+const { black, gray3 } = colors
+
+const Input = styled.input`
+  width: 100%;
+  padding: 1rem;
+  box-sizing: border-box;
+  border: 1px solid rgba(224, 224, 224, 0.5);
+  background: rgba(196, 196, 196, 0.05);
+  outline: none;
+  font-weight: 700;
+  font-size: ${size.xxs};
+  letter-spacing: ${letterSpacing.medium};
```

Comment:
This is much letter spacing than the Figam I think you would have to use **letterSpacing.small** I tried it and it was pretty closer to the Figma